doc/source/api/oxygen_xml.rst

```diff
@@ -1,7 +1,7 @@
 Oxygen XML tree
 ===============
 
-.. currentmodule:: pydita_ast.ast_tree
+.. currentmodule:: pyconverter.xml2py.ast_tree
 
 The following class is used to get an Oxygen XML tree:
```

Comment:
possible change "to get" to "to generate"?

Reply:
Sure! I'll do it